CoreWebServer/src/au/gov/asd/tac/constellation/webserver/WebServer.java

```diff
@@ -120,7 +120,6 @@ public static boolean checkSecret(final HttpServletRequest request, final HttpSe
     private static int port = 0;
 
     static final String CONSTELLATION_CLIENT = "constellation_client.py";
-    static final String RESOURCES = "resources/";
```

Comment:
Is there a reason why a constant isn't being used anymore?

Reply:
@antares1470 i think the issue was dealing with merge conflicts and some of your changes being accidentally overridden.